phpstan.neon.dist

```diff
@@ -39,3 +40,10 @@ parameters:
         - '/Class Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken constructor invoked with 4 parameters\, 2\-3 required./'
         - '/Method Symfony\\Component\\Serializer\\Normalizer\\NormalizerInterface\:\:supportsNormalization\(\) invoked with 3 parameters\, 1\-2 required\./'
         - '/Symfony\\Component\\Serializer\\NameConverter\\NameConverterInterface::normalize\(\) invoked with 2 parameters, 1 required./'
+
+    symfony:
+        containerXmlPath: var/cache/dev/App_KernelDevDebugContainer.xml
+
+    # Uncomment once we start using PHP configs, on Symfony 6+ it doesn't generate the below directory if no PHP config is present
```

Comment:
Sounds like this comment is for sylius 2.0, would it be better to create ticket for this and remove comment?